np2.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def np_random_vector():
+    """
+    Function that creates a vector of length 10 with values
+    evenly distributed between 5 and 50
+    :return: Vector
+    """
+    vector = np.random.randint(5, 51, 10)
```

Comment:
You were asked to evenly distribute (I think you understoof this as unformally distributed).
Look into numpy.linspace
https://numpy.org/doc/stable/reference/generated/numpy.linspace.html